test/files/gem003/generate.py

```diff
@@ -0,0 +1,46 @@
+# see toturial Kianwee Chen at https://academy.ifcopenshell.org/posts/creating-a-simple-wall-with-property-set-and-quantity-information/
+
+import ifcopenshell
+import ifcopenshell.template
+import itertools
+
+for num_body_identifier, num_axis_identifier in itertools.product(range(4), range(4)):
+    num_representations = num_body_identifier+num_axis_identifier
+    if not 1 <= num_body_identifier+num_axis_identifier <= 4:
+        continue
+    if num_axis_identifier > 2:
+        continue
+
+
+    file = ifcopenshell.template.create(schema_identifier="IFC2X3")
+    context = file.by_type("IfcGeometricRepresentationContext")[0]
+    owner = file.by_type("IfcOwnerHistory")[0]
+    building_parent = proj = file.by_type("IfcProject")[0]
+    site = file.createIfcSite(ifcopenshell.guid.new(), owner)
+
+    building = file.createIfcBuilding(
+                    ifcopenshell.guid.new(), owner
+                )
+
+    file.createIfcRelAggregates(
+            ifcopenshell.guid.new(),
+            owner,
+            RelatingObject=site,
+            RelatedObjects = [building]
+        )
+
+    create_ifcpolyline = lambda point_list : file.createIfcPolyLine([file.createIfcCartesianPoint(point) for point in point_list])
+    polyline = create_ifcpolyline([(0.0, 0.0, 0.0), (5.0, 0.0, 0.0)])
+
+    identifiers = ['Body' for i in range(num_body_identifier)] + ['Axis' for i in range(num_axis_identifier)]
+
+    representations = [file.createIfcShapeRepresentation(context, i, "Curve2D", [polyline]
```

Comment:
This creates a `Body` representation with `Curve2D` and IfcPolyline. I really prefer we have an IfcExtrudedAreaSolid here, so that the file makes a bit more sense.